lib/logstash/outputs/elasticsearch.rb

```diff
@@ -128,6 +128,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # a timeout occurs, the request will be retried.
   config :timeout, :validate => :number
 
+  # Set the Elasticsearch errors that you don't want to log. By default we log 
+  # every error. A useful example is when you want to skip all 409 errors
+  # which are `document_already_exists_exception`
+  config :skip_errors, :validate => :array, :default  => [""]
```

Comment:
that looks good.. however we will be skipping responses by status code rather than the elasticsearch error itself. At this point, we know that this is an answer from elasticsearch, and we have the `action_props`  object in `common.rb`. I think that we could skip ES errors than certain status codes. What do you think?